flows/download_catalog.py

```diff
@@ -294,62 +330,85 @@ def query_all(coo_centre, radius=24*u.arcmin, dist_cutoff=2*u.arcsec):
 		list: List of dicts with catalog stars.
 
 	.. codeauthor:: Rasmus Handberg <rasmush@phys.au.dk>
+	.. codeauthor:: Emir Karamehmetoglu <emir.k@phys.au.dk>
 	"""
 
 	# Query the REFCAT2 catalog using CasJobs around the target position:
 	results = query_casjobs_refcat2(coo_centre, radius=radius)
+	AT_results = Table(results)
```

Comment:
Tables or vectorized arrays are a much better way of working with data like this. It'll allow us to filter easily without having to do big for loops (your comment below). By converting to a table, we can get rid of the big for loop below. We can also keep track of rows of information about the star more easily in case we want to use errors, flags, etc. when selecting the catalog. All that info comes for free. You don't have to do line by line list comprehension. It's the preferred way of wroking with data like this, either Pandas dataframes or astropy tables.